ocs_ci/utility/azure_utils.py

```diff
@@ -309,3 +309,14 @@ def get_disk_state(self, volume_name):
 
         """
         return self.compute_client.disks.get(self.cluster_resource_group, volume_name).disk_state
+
+    def start_az_vm_instance(self, vm_name):
```

Comment:
`az` could be associated with availability zone and not azure. Please change the function name to `start_azure_vm_instance`

Reply:
Incorporated changes